cloudcix_primitives/snapshot_lxd.py

```diff
@@ -1,31 +1,552 @@
+        
+        # Default response if operations weren't properly tracked
+        if snapshot_name:
+            return True, data_dict, f'1200: {messages[1200]}'
+        else:
+            return True, data_dict, f'1201: {messages[1201]}'
+
+
+def scrub(
+        endpoint_url: str,
+        project: str,
+        instance_name: str,
+        instance_type: str,
+        snapshot_name: str,
+        verify_lxd_certs: bool = True
+) -> Tuple[bool, str, dict]:
+    """Delete a snapshot from an LXD instance.
+    
+    :param endpoint_url: The endpoint URL for the LXD Host.
+    :param project: The LXD project name.
+    :param instance_name: The name of the LXD instance containing the snapshot.
+    :param instance_type: The type of the LXD instance, either "containers" or "virtual_machines".
+    :param snapshot_name: The name of the snapshot to delete.
+    :param verify_lxd_certs: Boolean to verify LXD certs.
+    
+    :return: A tuple with a boolean flag indicating success or failure, a message, and a dictionary of successful payloads.
+    """
+    # Define messages
+    messages = {
+        1100: f'Successfully deleted snapshot {snapshot_name} from {instance_type} {instance_name} on {endpoint_url}',
+        3111: f'Invalid instance_type "{instance_type}" sent. Supported instance types are {", ".join(SUPPORTED_INSTANCES)}',
+        3121: f'Failed to connect to {endpoint_url} for {instance_type}.get payload',
+        3122: f'Failed to run {instance_type}.get payload on {endpoint_url}',
+        3123: f'Failed to find snapshot {snapshot_name} for {instance_type} {instance_name}',
+        3124: f'Failed to delete snapshot {snapshot_name}. Error: ',
+    }
+
+    # Validation
+    if instance_type not in SUPPORTED_INSTANCES:
+        return False, f'3111: {messages[3111]}', {}
+
+    def run_host(endpoint_url, prefix, successful_payloads):
+        rcc = LXDCommsWrapper(comms_lxd, endpoint_url, verify_lxd_certs, project)
+        fmt = HostErrorFormatter(
+            endpoint_url,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads,
+        )
+        
+        # Get the instance
+        ret = rcc.run(cli=f'{instance_type}.get', name=instance_name)
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f"{prefix+1}: {messages[prefix+1]}"), fmt.successful_payloads
+        if ret["payload_code"] != API_SUCCESS:
+            return False, fmt.payload_error(ret, f"{prefix+2}: {messages[prefix+2]}"), fmt.successful_payloads
+
+        instance = ret['payload_message']
+        fmt.add_successful(f'{instance_type}.get', ret)
+        
+        # Find and delete the snapshot
+        try:
+            # Get the snapshot
+            snapshot = instance.snapshots.get(snapshot_name)
+            if not snapshot:
+                return False, f"{prefix+3}: {messages[prefix+3]}", fmt.successful_payloads
```

Comment:
```suggestion
                return True, f'1101: {messages[1101]}', fmt.successful_payloads
```

If snapshot no longer exists this indicates it was scrubbed already so this can return True. `messages` will need to be extended for key `1101`

On line 415, the `msg` empty string can be changed to `f'1100: {messages[1100]}'. 

Then on line 419, the need for the if statement is no longer required and you can return the `status` and `msg` from the `run_host` function call